_tabs/principles.md

```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
 order: 0
 ---
 
-**We are a fun bunch of devs passionate about collaborating and building easy to use, valuable products. To do so, we abide by the following princilpes**
-
+**We are a bunch of passionate devs having fun building easy to use and valuable products together. To do so, we follow these princilpes**
 
 ## We reduce scope
-   - The best way to ship more stuff is to ship smaller!
-   - The best way to ship safely is to ship often!
+_"Focus Narrow, Deliver Big: Less really is more."_
+
+- What to do
+   - Breaking down large features into smaller, manageable chunks to ensure quicker delivery.
+   - Prioritizing must-have features over nice-to-have ones to focus on core functionality.
+   - Saying “no” to features that don't contribute to the main objective or delaying them for future iterations.
+
+- What to avoid
+   - Overcommitting to too many features at once.
+   - Spending time on polishing non-essential elements or perfecting UI components before validating the core.
+   - Failing to clearly define what’s in or out of scope, leading to scope creep.
 
 ## We ship early
-   - Getting feedback as early as possible even if the project/feature isn’t 100% there
-   - While we control radius with feature flags to reduce risk and deliver value to our customers
+_"Ship Sooner, Learn Faster: Early delivery for quicker feedback."_
+
+- What to do
+   - Releasing minimum viable products (MVPs) to get feedback from users quickly.
+   - Setting aggressive but realistic release targets to avoid endless development cycles.
+   - Prioritizing time-to-market over exhaustive development, even if it means the first version has known limitations.
+
+- What to avoid
+   - Waiting for a product to be "perfect" before releasing it.
+   - Ignoring early customer feedback or postponing releases due to minor issues.
```

Comment:
I don't follow how "ignoring early customer feedback" relates to "postponing releases due to minor issues". Also the latter kinda hints at the same thing as the "waiting for a product to be "perfect" before releasing it